src/components/Filters/ViewsMenu.tsx

```diff
@@ -105,7 +105,7 @@ const ViewsMenu = (props: ViewsMenuProps) => {
 
 	return (
 		<DropDownButton
-			ml={soloRender ? 0 : 30}
+			mx={soloRender ? 0 : 1}
```

Comment:
What's the point of `soloRender`? I see it's based on `renderMode` and `renderMode` is not used for anything except determining `soloRender`. Wouldn't it make more sense to remove this `ml`/`mx` from ViewsMenu and to instead apply it to ViewsMenu via extended FlexProps in the parent component?
i.e.:
```
<>
...
<ViewsMenu mx={renderMode?.length === 1 ? 0 : 1} {...otherRequiredProps} />
...
</>
```

Reply:
Sorry I haven't seen this comment yesterday. I would not expand ViewsMenu props since soloRender is used in two cases, the first is `alignRight` and the second is `mx`, so we should expand it with DropDownProps. I don't like the fact that Filters would contain a declaration that will be used only for ViewsMenu. This part needs a refactor, but I'll not do that in this PR. I've improved it, removing the entire if. I'm also wondering what is the case where we render only views... This button, without filters, is pretty useless IMHO...